egs/tedlium/s5/local/chain/README

```diff
@@ -0,0 +1,31 @@
+These are instruction to reproduce the TEDLIUM models described in
+"Purely sequence-trained neural networks for ASR based on lattice-free
+MMI", by Povey et al.
+
+First run:
+
+./run.sh
+
+until the end of stage 7. (local/nnet/run_dnn.sh can be skipped.)
+
+Then run:
+
+local/chain/run_tdnn.sh
+
+to see results for a generic chain model (these were not the results
+shown in the paper).
+
+Only after that, you can run:
+
+local/chain/run_tdnn_more_ce.sh
+
+to repeat the experiment reported in the third column of table 4.
+(The second script depends on files produced by the first one by
+default. It could be modified so that they are independent.) This
+experiment uses more cross entropy regularization and more parameters.
+
+Results for each setup are posted at the top of each file.
+
+The rescored test WERs are the same (12.9%), and the second experiment
```

Comment:
There are other reasons for variability e.g. dithering done during feature extraction of test data. Further even the ivector extractors are not the same across the runs. We acknowledge the fact that there is noise in the results generated across different runs. This is true for all experiments in all example recipes. So I don't see any reason for explicitly mentioning this here in particular.